docs/src/basic_concepts/record_fields.md

```diff
@@ -1,28 +1,22 @@
 ## Record organization
-The format uses various fields to organize and distinguish **records**
-and **record sets**[^1]:
-`name`, `uas_id` and `record_number`.
+The format uses different structures to organize and distinguish data.
```

Comment:
This doesn't say much, might as well remove it

Reply:
I agree that it doesn't provide much, but won't it feel a bit blunt without some sort of brief text?

Reply:
Then I suggest something like `most data is given as _records_, which combine with extra informationto create _record sets_`